.github/workflows/tests_and_release.yml

```diff
@@ -0,0 +1,68 @@
+name: Validate and Release connector
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Tableau Connector SDK
+        uses: actions/checkout@v4
+        with:
+          repository: tableau/connector-plugin-sdk
+          ref: tableau-2024.2
+
+      - name: Checkout CrateDB connector
+        uses: actions/checkout@v4
+        with:
+          path: cratedb-tableau-connector
```

Comment:
Is using a custom path needed / sensible?

Reply:
Ah, there are two artefacts needed here, so the path customization is here to discriminate between both? I get it now, it's fine! Thanks.

Reply:
Yes, otherwise one steps into the other https://github.com/actions/checkout/issues/33#issuecomment-595988890